src/app/productDetail/templates/productDetail.html

```diff
@@ -5,12 +5,16 @@
     <hr>
     <div class="row">
         <div class="col-md-6">
-            <!--PRODUCT IMAGE-->
-            <div class="thumbnail">
+            <!-- PRODUCT IMAGES -->            
+            <div class="thumbnail" style="max-width: 400px; height: auto; margin: 0 auto;" ng-if="!productDetail.item.xp.additionalImages || !productDetail.item.xp.additionalImages.length">
```

Comment:
If image zoom is off and there are no additional images, just show the thumbnail of the default image